crates/libcontainer/src/process/container_init_process.rs

```diff
@@ -270,6 +270,68 @@ fn reopen_dev_null() -> Result<()> {
     Ok(())
 }
 
+fn unmount_or_hide(syscall: &dyn Syscall, target: impl AsRef<Path>) -> Result<()> {
```

Comment:
Can  you add a doc comment for this function, explaining the intention behind it and use?